src/routes/relay/relay.controller.spec.ts

```diff
@@ -519,4 +519,10 @@ describe('RelayController', () => {
       });
     });
   });
+
+  describe('/v1/relay/health (GET)', () => {
+    it('should return a 200 response', async () => {
+      await request(app.getHttpServer()).get(`/v1/relay/health`).expect(200);
```

Comment:
nitpick: I would add another `.expect` for the body. Even though it's empty, it's valuable that we also test it. This prevents a situation where the a non-empty body gets added to the route (by mistake) and the test is successful.

```typescript
.expect(200)
.expect({});
```

Reply:
This can go in the same `expect` as well:

```ts
.expect(200, {})
```

Reply:
> This can go in the same `expect` as well:
> 
> ```ts
> .expect(200, {})
> ```

I prefer this version as the other one seems like an overload of the `expect` function and your versions is more clear that we are validating different things!